pandas/tests/plotting/test_converter.py

```diff
@@ -59,16 +60,15 @@ def test_register_by_default(self):
         call = [sys.executable, "-c", code]
         assert subprocess.check_call(call) == 0
 
+    @td.skip_if_low_mpl
```

Comment:
yes right idea 

i think we have some skips that are based on mpl versions already (maybe in the other test plotting files)

can u name like them

Reply:
emm, i see we have `skip_if_no_mpl` and `skip_if_mpl` decorators, but seems no version involved.

Reply:
does `skip_if_old_mpl` sound better? or `skip_if_lower_3_1_2_mpl`?

Reply:
latter